tests/Fixer/Whitespace/IndentationTypeFixerTest.php

```diff
@@ -324,4 +324,69 @@ static function ($key) {
             ARRAY_FILTER_USE_KEY
         );
     }
+
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideDoubleSpaceIndentCases
+     */
+    public function testDoubleSpaceIndent($expected, $input = null)
+    {
+        $this->fixer->setWhitespacesConfig(new WhitespacesFixerConfig('  '));
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideDoubleSpaceIndentCases()
+    {
+        return [
+            ['<?php
+if (true) {
+  if (true) {
+    (new stdClass())->foo(
+      "text",
+      "text2"
+    );
+  }
+}'],
+            [
+                "<?php
+if (true) {
+  if (true) {
+    (new stdClass())->foo(
+      'text',
+      'text2'
+    );
+  }
+}",
+                "<?php
+if (true) {
+  if (true) {
+\t(new stdClass())->foo(
+\t  'text',
+\t  'text2'
+\t);
+  }
+}",
+            ],
+            ['<?php
```

Comment:
This case is redundant with the subsequent one (as `doTest()` asserts that the expected part doesn't get changed by the fixer when used as input).

Reply:
:+1:  force pushed it out